README.md

```diff
@@ -1,5 +1,38 @@
+
+#User story
+As a movie enthusiast who wants to enjoy a meal while watching a movie,
+I want an Application that suggests restaurants based on location, customer rating, distance, price range,
+And also allows me to order food for delivery or pickup,
+So that I can conveniently order food and enjoy it while watching a movie.
+
+##Acceptance Criteria:
+- The application should have an integrated movie selection feature or integrate with popular movie streaming platforms.
+
+- It should provide a search feature to filter restaurants based on location, customer rating, distance, and price range.
+
+- The application should display a list of restaurants matching the search criteria, including relevant details like cuisine type, opening hours, and contact information.
+
+- It should have a user-friendly interface that allows me to easily navigate and browse restaurant options.
+
+- The application should integrate with popular food delivery or pickup services to facilitate seamless ordering.
+
+- It should provide a menu view for each restaurant, including individual dish details, ingredients, and price.
+
+- The application should have a secure and convenient payment system for online transactions.
+
+- It should offer options to customize the food order, such as selecting toppings, special requests, or dietary preferences.
+
+- The application should allow me to track the status of my food order i.e., cooking, preparing, out for delivery, etc.
+
+- It should notify me of the estimated delivery or pickup time.
+
+- The application should have a feature to rate and provide feedback on the delivered food and overall restaurant experience.
+
+- It should have a responsive customer support system to address any queries or issues related to ordering food or watching a movie simultaneously.
+=======
 # Food Picker 3000
 
 Sources:
 Emojis:
 Slot Machine animation:
+
```

Comment:
Awesome job on the PR. However, with this app we want it to pick a food item near the user, NOT order it for them. They do that on their own outside of the app.

Reply:
ohh ok i will modify  it